docs/user/04-metrics.md

```diff
@@ -626,13 +648,17 @@ To detect and fix such situations, check the pipeline status and check out [Trou
 2023-08-29T09:53:07.123Z warn internal/transaction.go:111 Failed to scrape Prometheus endpoint {"kind": "receiver", "name": "prometheus/app-pods", "data_type": "metrics", "scrape_timestamp": 1693302787120, "target_labels": "{__name__=\"up\", instance=\"10.42.0.18:8080\", job=\"app-pods\"}"}
 ```
 
-**Cause**: The workload is not configured to use 'STRICT' mTLS mode. For details, see [Activate Prometheus-based metrics](#4-activate-prometheus-based-metrics).
+**Cause 1**: The workload is not configured to use 'STRICT' mTLS mode. For details, see [Activate Prometheus-based metrics](#_4-activate-prometheus-based-metrics).
```

Comment:
```suggestion
**Cause 1**: The workload is not configured to use 'STRICT' mTLS mode. For details, see [Activate Prometheus-based metrics](#4-activate-prometheus-based-metrics).
```
I think the link is broken with underscore

Reply:
Without it is also broken... that's why I tried it with underscore

Reply:
Not sure how to fix it, canwe find a solution without link?

Reply:
arrrrgh :(

Reply:
After merging https://github.com/kyma-project/kyma/pull/18744, it should work without underscores.

Reply:
```suggestion
**Cause 1**: The workload is not configured to use 'STRICT' mTLS mode. For details, see [Activate Prometheus-based metrics](#step-4-activate-prometheus-based-metrics).
```

Reply:
as suggested in https://github.com/kyma-project/telemetry-manager/pull/1413#discussion_r1743699134, we might have to use html so that both, docsify and github, render it correctly.

Reply:
fixed